<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>720</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
   </layout>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>-1</y>
      <width>871</width>
      <height>701</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="CameraTab">
     <attribute name="title">
      <string>Camera</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="videoDisplayWidget">
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Choose the camera input below!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="readButton">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Start Connection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="databaseLbl">
        <property name="maximumSize">
         <size>
          <width>10000</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Choose Database and Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="databaseEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="cameraLbl">
        <property name="maximumSize">
         <size>
          <width>10000</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Choose Camera Input</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="CameraComboBox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="TableTab">
     <attribute name="title">
      <string>Database</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>21</y>
        <width>851</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="loadBtn">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>620</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Details</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="DetailsTab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Details</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>831</width>
        <height>611</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="captureDisplayWidget">
         <property name="text">
          <string>Detection Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="MapWebView">
         <property name="text">
          <string>MapWebView</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="latitudeLbl">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Latitude</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="latitudeEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="altitudeLbl">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Altitude</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="altitudeEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="longitudeLbl">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Longitude</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="longitudeEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="updateBtn">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
